nifi-frontend/src/main/frontend/apps/nifi/src/app/app.module.ts

```diff
@@ -51,6 +55,14 @@ import { loadingInterceptor } from './service/interceptors/loading.interceptor';
 import { LoginConfigurationEffects } from './state/login-configuration/login-configuration.effects';
 import { BannerTextEffects } from './state/banner-text/banner-text.effects';
 
+const entry = localStorage.getItem('disable-animations');
+let disableAnimations: string = entry !== null ? JSON.parse(entry).item : '';
+
+// honor OS settings if user has not explicitly disabled animations for the application
+if (disableAnimations !== 'true' && disableAnimations !== 'false') {
+    disableAnimations = window.matchMedia('(prefers-reduced-motion: reduce)').matches.toString();
```

Comment:
In the navigation component, we ensure that `matchMedia` is available because attempting to use it. Is that check necessary here?

Reply:
I think we can get rid of it in the navigation component.